File: 6KYU/Sort the Odd/solution.cs
using System.Linq;
public class Kata
{
  public static int[] SortArray(int[] input)
  {
      var odds = input.Where(n => n % 2 != 0).OrderBy(c => c).ToArray();
      var oddCounter = 0;
      for (var i = 0; i < input.Length; i++)
      {
        if (input[i] % 2 != 0)
        {
          input[i] = odds[oddCounter];
          oddCounter++;
        }
      }
      return input;
  }
}

File: 6KYU/Sort the Odd/solution.py
def sort_array(xs):
    odds = sorted([i for i in xs if i % 2 != 0])
    odd_counter = 0
    res = []
    index = 0
    for i in xs:
        if i % 2 != 0:
            res.append(odds[odd_counter])
            odd_counter+=1
        else:
            res.append(i)
        index+=1
    return res